etc/profile-m-z/weechat.profile

```diff
@@ -11,6 +11,8 @@ noblacklist ${HOME}/.weechat
 include disable-common.inc
 include disable-programs.inc
 
+whitelist /usr/share/weechat
+
```

Comment:
```suggestion
```

Reply:
imho the whitespace helps separate the logical blocks here, but this is not the right place to discuss this